wandb/sdk/wandb_init.py

```diff
@@ -490,6 +497,12 @@ def _make_run_disabled(self) -> RunDisabled:
         )
         return drun
 
+    def _on_init_progress(self, handle: MailboxHandle) -> None:
+        assert self.printer
+        line = "Waiting for wandb.init()...\r"
```

Comment:
Shall we report what `settings.init_timeout` is set to in absolute seconds? Just thinking about how to make it more discoverable.

Reply:
agreed, that is planned for phase2 in the PR description.